stdlib/public/core/LifetimeManager.swift

```diff
@@ -283,3 +283,40 @@ extension String {
 public func _copy<T>(_ value: T) -> T {
   copy value
 }
+
+/// Unsafely discard any lifetime dependency on the `dependent` argument. Return
+/// a value identical to `dependent` with a lifetime dependency on the caller's
+/// borrow scope of the `borrows` argument.
+///
+/// TODO: Remove @_unsafeNonescapableResult. Instead, the unsafe dependence
+/// should be expressed by a builtin that is hidden within the function body.
+@_unsafeNonescapableResult
+@_alwaysEmitIntoClient
+@inline(__always)
+@lifetime(borrow source)
+internal func _unsafeLifetime<T: ~Copyable & ~Escapable,
+                              U: ~Copyable & ~Escapable>(
+  dependent: consuming T, borrows source: borrowing U
+) -> T {
+  dependent
+}
+
+/// Unsafely discard any lifetime dependency on the `dependent` argument. Return
+/// a value identical to `dependent` that inherits all lifetime dependencies from
+/// the `dependsOn` argument.
+///
+/// Transferring generic lifetime dependencies from `dependsOn` requires the
+/// result to have the same type.
```

Comment:
```suggestion
/// Transferring generic lifetime dependencies from `source` requires the
/// result to have the same type.
```

I can't really understand this sentence. What does it mean in practice?

Reply:
Deleted. That sentence no longer applies to the current form.